pkg/offline/offline.go

```diff
@@ -390,3 +413,14 @@ func (q *Queue) PushMany(hh []heartbeat.Heartbeat) error {
 
 	return nil
 }
+
+// Count returns the total number of heartbeats in the offline db.
+func (q *Queue) Count() (int, error) {
+	b, err := q.tx.CreateBucketIfNotExists([]byte(q.Bucket))
+	if err != nil {
+		log.Errorf("failed to create/load bucket: %s", err)
+		return 0, fmt.Errorf("failed to create/load bucket: %s", err)
```

Comment:
Instead of duplicating error message here it could only return and the error would be checked on the caller.